sites.json

```diff
@@ -3791,6 +3791,17 @@
             "lovefilm.com"
         ]
     },
+    
+    {
+		"name": "Lucidchart/Lucidpress",
```

Comment:
Do these sites share an account system? That is, is a Lucidchart login valid for the Lucidpress site as well? And would deletion of that (presumably shared) login remove access to both sites?

Reply:
All questions can be answered in the affirmative